app/src/main/res/layout/fragment_shopinfo.xml

```diff
@@ -48,62 +49,105 @@
+                        android:layout_height="@dimen/dp_120"
+                        android:layout_alignParentBottom="true"
+                        android:orientation="vertical"
+                        tools:ignore="UselessParent"
+                        >
+
+                        <ImageView
+                            android:layout_width="match_parent"
+                            android:layout_height="@dimen/dp_50"
+                            android:layout_alignParentBottom="true"
+                            android:background="@color/color_white_transparent"
+                            tools:ignore="ContentDescription,UselessParent"
+                            />
+                        <ImageView
+                            android:id="@+id/image_shop"
+                            android:layout_width="@dimen/dp_120"
+                            android:layout_height="@dimen/dp_120"
+                            android:layout_alignParentTop="true"
+                            android:layout_centerHorizontal="true"
+                            android:background="@drawable/border_comment"
+                            android:contentDescription="@string/avatar_shop"
+                            android:scaleType="fitXY"
+                            app:imageUrl="@{viewModel.shopAvatar}"
+                            />
+                    </RelativeLayout>
+                </RelativeLayout>
+            </android.support.design.widget.CollapsingToolbarLayout>
+
+        </android.support.design.widget.AppBarLayout>
+
+        <android.support.v4.widget.NestedScrollView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@android:color/white"
+            app:layout_behavior="@string/appbar_scrolling_view_behavior"
+            >
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/dp_10"
+                android:orientation="vertical"
+                >
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:background="@color/color_white"
+                    android:orientation="vertical"
                     >
-                    <!--Name Shop-->
                     <TextView
                         android:id="@id/text_shop_name"
                         android:layout_width="match_parent"
                         android:layout_height="wrap_content"
                         android:layout_alignParentBottom="true"
-                        android:background="@color/color_white_transparent"
                         android:gravity="center"
-                        android:textColor="@color/color_orange_mos"
+                        android:paddingLeft="@dimen/dp_10"
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingLeft` you should probably also define `paddingRight` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.